src/start/CompatibilityScreen.js

```diff
@@ -15,17 +17,58 @@ const styles = StyleSheet.create({
     color: 'white',
     fontSize: 20,
     margin: 8,
-  }
+  },
+  touchTarget: {
+    flexDirection: 'row',
+    alignSelf: 'stretch',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  logo: {
+    width: 200,
+    height: 100,
+  },
 });
 
-export default class CompatibilityScreen extends React.Component {
 
+export default class CompatibilityScreen extends React.Component {
+  handleClick = (link: string) => {
+    Linking.canOpenURL(link).then(supported => {
+      if (!supported) {
+        console.log('Can\'t open URL: ', link);
```

Comment:
Maybe show a modal alert here that says something like 'Could not open app store.'